src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Macros/GuidMacro.cs

```diff
@@ -70,24 +72,27 @@ public void EvaluateConfig(IEngineEnvironmentSettings environmentSettings, IVari
                 };
             }
 
-            vars[config.VariableName] = g.ToString(config.DefaultFormat);
-            setter(pd, g.ToString(config.DefaultFormat));
+            string defaultValue = char.IsUpper(config.DefaultFormat[0]) ?
```

Comment:
We should probably check if `DefaultFormat` is not empty string.

Reply:
In theory that is already done in config class... Want me to add here too?

Reply:
I will leave it up to you.